entire_vds_pipeline.py

```diff
@@ -414,8 +416,28 @@ def export_to_elasticsearch(
 logger.info("\n==> create HailContext")
 hc = hail.HailContext(log="/hail.log")
 
+logger.info("Reading in dataset...")
 vds = read_in_dataset(input_path, args.datatype, filter_interval)
 
+#NOTE: if sample IDs are remapped first thing, then the fam file should contain the desired (not original IDs)
+if args.remap_sample_ids:
+    logger.info("Remapping sample ids...")
+    id_map = hc.import_table(args.remap_sample_ids, impute=True, no_header=True)
+    mapping = dict(zip(id_map.query('f0.collect()'), id_map.query('f1.collect()')))
+    vds = vds.rename_samples(mapping)
```

Comment:
would be good to check that all keys in the dictionary match sample_ids in the VDS and print a warning, or halt (maybe depending on whether --ignore-extra-sample-ids-in-fam-file was used - though this arg can now be renamed to something like --ignore-extra-sample-ids-in-tables).